ConsolePhoneBookApp/BusinessLogic/Contact.cs

```diff
@@ -1,31 +1,39 @@
-﻿using System;
+﻿using BusinessLogic.Interfaces;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace BusinessLogic
 {
-    public class Contact
+    public class Contact: IContact    
     {
-        private string _name;
-        private string _number;
-    
-        public string Name
-        {
-            get { return _name; }
-            set { _name = value; }
```

Comment:
private set

Reply:
public string Name { get; set; } > private set

In controller put name using Constructor 

remove public Contact() { } - useless